echoserver/lib/echoserver.ex

```diff
@@ -0,0 +1,19 @@
+defmodule EchoServer do
+  use GenServer
+
+  @moduledoc """
+  Модуль простого эхо сервера.
+  """
+
+  def start_link(opts \\ []) do
+    GenServer.start_link(__MODULE__, [], opts)
+  end
+
+  def init(_) do
+    {:ok, nil}
```

Comment:
Это не имеет большого практического смысла, но лучше не плодить сущности: вам не нужен стейт (что правильно), но сначала вы намекаете, что он будет `[]`, а потом ниже выбрасываете это значение и устанавливаете его в `nil`. Лучше выбрать что-то одно и в `init/1` ничего не менять.
```suggestion
    GenServer.start_link(__MODULE__, nil, opts)
  end

  def init(init_arg) do
    {:ok, init_arg}
```